be/src/exec/tablet_sink.cpp

```diff
@@ -845,40 +845,35 @@ Status OlapTableSink::close(RuntimeState* state, Status close_status) {
 }
 
 void OlapTableSink::_print_varchar_error_msg(RuntimeState* state, const Slice& str, SlotDescriptor* desc) {
-    std::stringstream ss;
-    ss << "the length of input is too long than schema. "
-       << "column_name: " << desc->col_name() << "; "
-       << "input_str: [" << str.to_string() << "] "
-       << "schema length: " << desc->type().len << "; "
-       << "actual length: " << str.size << "; ";
+    std::string error_msg = strings::Substitute("String '$0' is too long. The type of '$1' is $2'", str.to_string(),
+                                                desc->col_name(), desc->type()->debug_string());
 #if BE_TEST
-    LOG(INFO) << ss.str();
+    LOG(INFO) << error_smg;
 #else
-    state->append_error_msg_to_file("", ss.str());
+    state->append_error_msg_to_file("", error_msg);
 #endif
 }
 
 void OlapTableSink::_print_decimal_error_msg(RuntimeState* state, const DecimalV2Value& decimal, SlotDescriptor* desc) {
-    std::stringstream ss;
-    ss << "decimal value is not valid for definition, column=" << desc->col_name() << ", value=" << decimal.to_string()
-       << ", precision=" << desc->type().precision << ", scale=" << desc->type().scale;
+    std::string error_msg = strings::Substitute("Decimal '$0' is out of range. The type of '$1' is $2'", decimal_str,
```

Comment:
better to tell the user what the specific value range is.

Reply:
The printed type will indicate the value range.